administrative-sdk/speechRecording.js

```diff
@@ -123,197 +102,155 @@ class SpeechRecording {
    *
    * @param {its.SpeechChallenge} challenge The pronunciation challenge to perform.
    * @param {its.AudioRecorder} recorder The audio recorder to extract audio from.
-   * @param {Sdk~speechRecordingPreparedCallback} [preparedCb] The callback that signals server is prepared for receiving data.
-   * @param {Sdk~speechRecordingCreatedCallback} [cb] The callback that handles the response. The success outcome is returned as first parameter, whether the recording was forcedStopped due to timer timeout is returned as second parameter.
+   * @param {Sdk~speechRecordingPreparedCallback} [preparedCb] The callback that signals server is prepared for
+   *   receiving data.
+   * @param {Sdk~speechRecordingCreatedCallback} [cb] The callback that handles the response. The success outcome is
+   *   returned as first parameter, whether the recording was forcedStopped due to timer timeout is returned as second
+   *   parameter.
    * @param {Sdk~speechRecordingCreatedErrorCallback} [ecb] The callback that handles the error response.
```

Comment:
Missing documentation update. Callbacks are no longer taken and a promise is returned.